bitnami/minio/README.md

```diff
@@ -128,6 +128,7 @@ The following table lists the configurable parameters of the MinIO chart and the
 | `ingress.enabled`                    | Enable/disable ingress                                                                                                                                    | `false`                                                 |
 | `ingress.certManager`                | Add annotations for cert-manager                                                                                                                          | `false`                                                 |
 | `ingress.annotations`                | Ingress annotations                                                                                                                                       | `[]`                                                    |
+| `ingress.labels`                     | Ingress labels                                                                                                                                            | `{}`                                                    |
```

Comment:
I would make it clear that these are _Additional_ labels, as Ingress will already have some other labels by default

https://github.com/bitnami/charts/blob/1a4419e15d985f67413beff98c9fc9b9f69108fb/bitnami/minio/templates/ingress.yaml#L7